### PR TITLE
Update DB Scripts

### PR DESCRIPTION
**Pull request by woogerboy21**
Added backup / restore scripts.
Added table size script
Updated existing scripts to use variable location for mysql.cnf file

### CODE REVIEW

**Comment by woogerboy21** [comment]
These additional scripts should help new servatrice operators backup / restore and maintain there databases.  The backup / restore scripts offer the ability to backup / restore at a table level.  So db tables can be ignored if desired.

**Comment by woogerboy21** [comment]
Hmm... I probably create windows scripts as well.  That will be a different PR later

File: servatrice/scripts/db_backup_all [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```
Comment (Daenyth) [inline servatrice/scripts/db_backup_all]: Can you `set -e` below this?

File: servatrice/scripts/db_restore_all [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+SLEEPTIME=5
```
Comment (Daenyth) [inline servatrice/scripts/db_restore_all]: `set -e` and `set -u` here also

File: servatrice/scripts/db_restore_all [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	"cockatrice_games_players"
+	"cockatrice_uptime"
+	"cockatrice_schema_version"
+	"cockatrice_servermessages"
+	"cockatrice_servers"
+	"cockatrice_news"
+	"cockatrice_rooms"
+        "cockatrice_rooms_gametypes"
+	)
+/home/wwadmin/scripts/db_maint_logs
```
Comment (Daenyth) [inline servatrice/scripts/db_restore_all]: This path isn't likely to be right.

File: servatrice/scripts/db_restore_all [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+if [ -z "$COMMANDRESULTS" ];
+then
+	[ ! -d $BACKUPDIR ] && mkdir -p $BACKUPDIR/
+	for TABLENAME in "${TABLES[@]}"
+	do
+		BACKUPFILE="$BACKUPDIR/$APPNAME.$TABLENAME.sql.$LOGAPPENDDATE"
+		if [ -f "$BACKUPFILE" ]
+		then
+			echo "Restoring up DB Table [$TABLENAME]"
+			ionice -c3 nice -n19 mysql --defaults-file=$SQLCONFFILE $DBNAME < $BACKUPFILE
```
Comment (Daenyth) [inline servatrice/scripts/db_restore_all]: What if conf doesn't exist?

File: servatrice/scripts/db_backup_all (hunk 1 above)
Comment (woogerboy21) [inline servatrice/scripts/db_backup_all]: I can do the -u option, but the -e option will always fail as the script runs the ps command looking for a command result and should always return a non-zero result if there is something running.  In the case that it does return that results the error displayed will no be presented and not inform the user why the script aborts. (note: you can't see it in this revision but you will see it in the next revision of the PR shortly).

File: servatrice/scripts/db_restore_all (hunk 2 above)
Comment (woogerboy21) [inline servatrice/scripts/db_restore_all]: I can do the -u option, but the -e option will always fail as the script runs the ps command looking for a command result and should always return a non-zero result if there is something running.  In the case that it does return that results the error displayed will no be presented and not inform the user why the script aborts. (note: you can't see it in this revision but you will see it in the next revision of the PR shortly).

File: servatrice/scripts/db_restore_all (hunk 3 above)
Comment (woogerboy21) [inline servatrice/scripts/db_restore_all]: Yeah this was a whoops, missed it when sanitizing the scripts for the PR.

File: servatrice/scripts/db_restore_all (hunk 4 above)
Comment (woogerboy21) [inline servatrice/scripts/db_restore_all]: The screen is outputed the following:
```
Backing up DB Table [cockatrice_users]
Could not open required defaults file: /tmp/mysql.cnf
```
(Or restore in place of backup depending on the script used).

File: servatrice/scripts/db_backup_all (hunk 1 above)
Comment (woogerboy21) [inline servatrice/scripts/db_backup_all]: Fixed the -e thing.

**Comment by woogerboy21** [comment]
@Daenyth anything else on this?  Im not sure why the previous comments that you made and I answered and have added the requested features to have not collapsed.

**Pull request merged**